easypermissions/src/main/java/pub/devrel/easypermissions/EasyPermissions.java

```diff
@@ -125,11 +125,13 @@ public static void requestPermissions(@NonNull final Object object, @NonNull Str
      *         request code to track this request, must be < 256.
      * @param perms
      *         a set of permissions to be requested.
+     * @return true if a permissions request was started, false otherwise.
      */
     @SuppressLint("NewApi")
-    public static void requestPermissions(@NonNull final Object object, @NonNull String rationale,
+    public static boolean requestPermissions(@NonNull final Object object, @NonNull String rationale,
```

Comment:
The essence of #62 was tracking all the technical debt that has built up over time.  This library made some bad initial decisions (using `Object` all over the place rather than overloads) and with the complexity of the support library there are just a ton of branches and checks that I'd like to refactor to make the code overall smaller and more readable.